python/src/trezorlib/cli/trezorctl.py

```diff
@@ -412,6 +412,12 @@ def set_u2f_counter(connect, counter):
     return device.set_u2f_counter(connect(), counter)
 
 
+@cli.command(help="Get U2F counter.")
+@click.pass_obj
+def get_u2f_counter(connect):
```

Comment:
Just a nitpick - shouldn't this follow the suit and be named `get_next_u2f_counter`?

Reply:
It should. Fixed